src/optimalcontrol/tests/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.1)
+
+set (CMAKE_CXX_STANDARD 11)
+
+
+project(OptimalControlTests)
+
+#add_executable(OptimalControlTest OptimalControlTest.cpp)
+
+#target_link_libraries(OptimalControlTest iDynTree::optimalcontrol ${iDynTree_LIBRARIES})
+
+#add_test(NAME OptimalControlTest
+#        COMMAND ${PROJECT_NAME})
+
+add_executable(ConstraintsGroupTest ConstraintsGroupTest.cpp)
+
+target_link_libraries(ConstraintsGroupTest PRIVATE idyntree-optimalcontrol ${iDynTree_LIBRARIES})
+
+
+add_test(NAME ConstraintsGroupTest
```

Comment:
Again a low priority comment, but we should follow the same scheme of the tests of the other components and also enabled Valgrind-based tests.